### DISCUSSION TITLE
Weblate - ECS - Postgres restoring

### DISCUSSION THREAD

**Discussion by AKIvan** [WeblateOrg/weblate#9668] [Q&A]
Hi all, 

I'm trying to migrate weblate from one Cloud Provider to another and this includes migration of the DB for the projects. Now during some local and cloud test i successfully did it (I have no idea what went wrong). 

Im trying to run weblate on ECS and im using one task with two containers "Weblate" and "Redis". Postgres is run as RDS. When i run the deployment fresh (without restoring the DB) everything is working ok, we have weblate up and running, fresh. 

Now i have test this with docker-compose where i run first postgres, make the restoring, and than run the weblate. And it works i have the previous projects and everything. 

So im trying to reproduce the same scenario on ecs, create first the RDS, make the restoring, and run the weblate, (again this was successfull scenario), for some reason im getting the following error:

`Restoring PostgresSQL - CommandError: Multiple users matched given parameters!`

And after some time it loose the connection to RDS. 

I'm using the database.sql file found in the backup directory. 
Any idea what could go wrong. I try using same users, same db names and everything still same problem. Any idea what im missing ?

**Comment by nijel** [comment]
Adjust WEBLATE_ADMIN_EMAIL setting to match what was on the previous server (and is stored in your database backup).

**Reply by AKIvan** [reply]
Yup that was it. Some of those env were not the same. 
Thanks !
